src/pages/signup-affiliates/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import Signup, { Appearances } from 'components/custom/signup'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+import { SEO } from 'components/containers'
+import device from 'themes/device.js'
+import { Header, Text } from 'components/elements'
+import Map from 'images/svg/landing/map.svg'
+
+const Wrapper = styled.section`
+    width: 100%;
+    height: 40vh;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    justify-content: center;
+    background-color: rgba(200, 214, 215, 0.22);
+
+    @media ${device.mobileL} {
+        padding: 0;
+    }
+`
+const Content = styled.div`
+    width: 42rem;
```

Comment:
I noticed you are specifying specific widths in a lot f these containers. Any particular reason we need to do this instead of using a parent flex and distributing them accordingly to their layouts? Specific widths like these will break down for smaller screens as this width will be larger than the smallest supported screen of 320px.

Reply:
I updated every own created flex-container with inner Flex components. Thanks.